Makefile

```diff
@@ -12,7 +27,7 @@ $(GENSRC): $(OPENAPIDEPS)
 
 # this exists to override pattern match rule above since this file is in the generated directory but should not be treated as generated code
 pkg/generated/restapi/configure_rekor_server.go: $(OPENAPIDEPS)
-	
+
```

Comment:
There needs to be a tab here for the Makefile to be correct

Reply:
it works locally for me as is
```
$ make pkg/generated/restapi/configure_rekor_server.go
make: `pkg/generated/restapi/configure_rekor_server.go' is up to date.

```

Reply:
but added back

Reply:
thanks for your review @bobcallaway